hw/xbox/nv2a/gl/gloffscreen_wgl.c

```diff
@@ -187,8 +181,9 @@ GloContext *glo_context_create(void) {
 }
 
 /* Set current context */
-void glo_set_current(GloContext *context) {
-
+void glo_set_current(GloContext *context)
+{
+    epoxy_handle_external_wglMakeCurrent();
```

Comment:
Note: This is the hack for libepoxy which might have performance impact (aside from libepoxy not being well designed, and possibly making *all* `wglMakeCurrent` generally slower than GLEW)